src/connector/src/parser/unified/json.rs

```diff
@@ -533,7 +539,7 @@ impl JsonParseOptions {
                                 }
                                 &BorrowedValue::Static(simd_json::StaticNode::Null)
                             });
-                        self.parse(field_value, field_type)
+                        self.parse(field_value, field_type).map(|d| d.to_owned_datum())
```

Comment:
Since the nexmark benchmark takes a struct as input, while it's not possible to achieve zero-copy when parsing a `Struct` field (yet), the performance gain in this PR may not be reflected.

Reply:
Do you think it's possible?

Reply:
Need further investigation. 😕